tests/Model/Generated/Object/NotificationUrlTest.php

```diff
@@ -0,0 +1,282 @@
+ *  NotificationUrl
+ */
+class NotificationUrlTest extends BunqSdkTestBase
+{
+    /**
+     * Getter string constants
+     */
+    const GET_PAYMENT = 'getPayment';
+    const GET_BUNQ_ME_TAB = 'getBunqMeTab';
+    const GET_CHAT_MESSAGE_ANNOUNCEMENT = 'getChatMessageAnnouncement';
+    const GET_DRAFT_PAYMENT = 'getDraftPayment';
+    const GET_MASTER_CARD_ACTION = 'getMasterCardAction';
+    const GET_MONETARY_ACCOUNT_BANK = 'getMonetaryAccountBank';
+    const GET_PAYMENT_BATCH = 'getPaymentBatch';
+    const GET_REQUEST_INQUIRY = 'getRequestInquiry';
+    const GET_REQUEST_RESPONSE = 'getRequestResponse';
+    const GET_SCHEDULE_PAYMENT = 'getScheduledPayment';
+    const GET_SCHEDULE_INSTANCE = 'getScheduledInstance';
+    const GET_SHARE_INVITE_BANK_INQUIRY = 'getShareInviteBankInquiry';
+    const GET_SHARE_INVITE_BANK_RESPONSE = 'getShareInviteBankResponse';
+
+    /**
+     * Assertion errors.
+     */
+    const ASSERT_SHOULD_NOT_REACH_THIS_CODE_ERROR = 'Something super weird just happen';
+    const ASSERT_JSON_DECODE_ERROR = 'Might be that the JSON file is not a valid json.';
+    const ASSERT_OBJECT_IS_NULL_ERROR = 'Object seems to be null.';
+
+
+    /**
+     * Model json paths constants.
+     */
+    const BASE_PATH_JSON_MODEL =  __DIR__ . '/../../../resource/NotificationUrlJsons';
+    const JSON_PATH_MUTATION_MODEL = self::BASE_PATH_JSON_MODEL . '/Mutation.json';
+    const JSON_PATH_BUNQ_ME_TAB_MODEL = self::BASE_PATH_JSON_MODEL . '/BunqMeTab.json';
+    const JSON_PATH_CHAT_MESSAGE_ANNOUNCEMENT_MODEL = self::BASE_PATH_JSON_MODEL . '/ChatMessageAnnouncement.json';
+    const JSON_PATH_DRAFT_PAYMENT_MODEL = self::BASE_PATH_JSON_MODEL . '/DraftPayment.json';
+    const JSON_PATH_MASTER_CARD_ACTION_MODEL = self::BASE_PATH_JSON_MODEL . '/MasterCardAction.json';
+    const JSON_PATH_MONETARY_ACCOUNT_BANK_MODEL = self::BASE_PATH_JSON_MODEL . '/MonetaryAccountBank.json';
+    const JSON_PATH_PAYMENT_BATCH_MODEL = self::BASE_PATH_JSON_MODEL . '/PaymentBatch.json';
+    const JSON_PATH_REQUEST_INQUIRY_MODEL = self::BASE_PATH_JSON_MODEL . '/RequestInquiry.json';
+    const JSON_PATH_REQUEST_RESPONSE_MODEL = self::BASE_PATH_JSON_MODEL . '/RequestResponse.json';
+    const JSON_PATH_SCHEDULE_PAYMENT_MODEL = self::BASE_PATH_JSON_MODEL . '/ScheduledPayment.json';
+    const JSON_PATH_SCHEDULE_INSTANCE_MODEL = self::BASE_PATH_JSON_MODEL . '/ScheduledInstance.json';
+    const JSON_PATH_SHARE_INVITE_BANK_INQUIRY_MODEL = self::BASE_PATH_JSON_MODEL . '/ShareInviteBankInquiry.json';
+    const JSON_PATH_SHARE_INVITE_BANK_RESPONSE_MODEL = self::BASE_PATH_JSON_MODEL . '/ShareInviteBankResponse.json';
+
+    /**
+     * Model root key.
+     */
+    const KEY_NOTIFICATION_URL_MODEL = 'NotificationUrl';
+    const KEY_MUTATION_MODEL = '';
+
+    /**
+     * @param string $fileName
+     * @param string $className
+     * @param string $getterName
+     */
+    private function executeTest(string $fileName, string $className, string $getterName)
```

Comment:
I'd say: `executeNotificationUrlTest` and `expectedJsonFileName`, `classNameExpected` and `referencedObjectGetterName`.